app/components/carousel/constants.ts

```diff
@@ -1,14 +1,16 @@
 import { Dimensions } from "react-native"
-import { spacing } from "../../theme"
+import { layout, spacing } from "../../theme"
 
-export const { width: SCREEN_WIDTH } = Dimensions.get("screen")
-// spacing.large is size of the content gutter for the app.
-export const SCREEN_CONTENT_GUTTER = spacing.large
-export const SCREEN_CONTENT_WIDTH = SCREEN_WIDTH - SCREEN_CONTENT_GUTTER * 2
-export const CAROUSEL_CARD_WIDTH = Math.floor(SCREEN_CONTENT_WIDTH * 0.95)
+// Note: This will not react to screen dimension changes post intial render
+const { width: screenWidth } = Dimensions.get("screen")
```

Comment:
I'd love to move these magic numbers into the carousel at some point. It was beyond the scope of this PR though.